exporters/scss/src/js/generators/typography.ts

```diff
@@ -32,15 +31,25 @@ export function generateTypography(
       }
     });
 
-    const subfamily = token.value.font.subfamily.toLowerCase();
     const fontSize = printUnit(Math.round((token.value.fontSize.measure / defaultFontSize) * 1000) / 1000, 'rem');
     let fontStyle = 'normal';
-    let fontWeight = 'normal';
-    if (subfamily === 'italic') {
+    let fontWeight = +token.value.font.subfamily;
+
+    // Font Ebony has a different font weight mapping, so we remap these values directly
+    if (token.value.font.family === 'Ebony') {
+      if (fontWeight === 400) {
+        fontWeight = 300;
+      }
+      if (fontWeight === 600) {
+        fontWeight = 400;
+      }
+    }
+
+    // TODO: This is a hack to get around the fact that I don't know how to check if font is italic in JS
```

Comment:
I think it's OK 🤷🏻‍♂️

Reply:
It is not, because you might have a typography style something like `signature` which is italic and we won't know about it. Hopefully, Supernova is going to help me how to do it.